src/pages/Question/Question.svelte

```diff
@@ -1 +1,39 @@
-<h1>question</h1>
+<script lang="ts">
+	// import Router from 'svelte-spa-router'
+import { QuestionCard } from '../../components'
+</script>
+
+
+<div style="min-height: 4rem"></div>
+<h1>Current questions</h1>
+<div class="questions_grid">
+    <QuestionCard level = {1} title='Question title' description='do hello world'/>
+    <QuestionCard level = {2} title='Question title' description='do hello world'/>
+    <QuestionCard level = {3} title='Question title' description='do hello world'/>
+    <QuestionCard level = {3} title='Question title' description='do hello world'/>
+</div>
+<h1>Past questions</h1>
+<div class="questions_grid">
```

Comment:
Can you make a QuestionsWrap or QuestionsGrid component that takes list of questions as a parameter? This will let us reuse the .questions_grid style in various different pages. 

For the "list of questions parameter" just assume it's a list with the correct info for now. Will eventually add a "Question" type, but for now just assume the data is fine.